src/modules/map/components/LocationPopup.vue

```diff
@@ -231,14 +230,14 @@ export default {
         },
         async updateHeight(coordinate) {
             try {
-                this.height = await requestHeight(coordinate)
+                this.height = await requestHeight(coordinate, this.projection)
             } catch (error) {
                 log.error(`Failed to get position height`)
                 this.height = null
             }
         },
         updateShareLink(coordinate, routeQuery) {
-            let [lon, lat] = reproject(this.projection.epsg, WGS84.epsg, coordinate)
+            let [lon, lat] = proj4(this.projection.epsg, WGS84.epsg, coordinate)
             let query = {
                 ...routeQuery,
                 crosshair: 'marker',
```

Comment:
This with your #463 PR won't be anymore valid, maybe rebase this on top of the other and use center instead of lon lat.